apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -55,3 +56,22 @@ val <T : Path> Expect<T>.parent get(): Expect<Path> = ExpectImpl.path.parent(thi
  */
 fun <T : Path> Expect<T>.parent(assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
     ExpectImpl.path.parent(this).addToInitial(assertionCreator)
+
+/**
+ * Creates an [Expect] after calling [Path.fileNameWithoutExtension] on the subject of the assertion,
+ * so that further fluent calls are assertions about it.
+ *
+ * @return The newly created [Expect].
+ */
+val <T : Path> Expect<T>.fileNameWithoutExtension
+    get(): Expect<String> = ExpectImpl.path.fileNameWithoutExtension(this).getExpectOfFeature()
+
+/**
+ * Expects that the result of [Path.fileNameWithoutExtension] on the subject of the assertion should
```

Comment:
Please copy the KDoc from an existing assertion function without too many modifications (make overall changes in the future simpler). I suggest to use something like:
```
/**
 * Expects that the property [Path.fileNameWithoutExtension] (provided via [niok](https://github.com/robstoll/niok)) 
 * of the subject of the assertion holds all assertions the given [assertionCreator] creates for it 
 * and returns this assertion container.
 *
 * @return This assertion container to support a fluent API.
 * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
 *
 * @since 0.9.0
 */
```